instructor/compiler.py

```diff
@@ -353,10 +353,10 @@ def drugs(assignment):
 def concentrations(assignment):
     ret = {}
     for strain_protocol in assignment.strain_treatment.filter(enabled=True):
-        treatment = strain_protocol.treatment
-        concentration = treatment.drug.concentration
-        ret[str(concentration)] = {
-            'name': str(concentration),
+        drug = strain_protocol.treatment.drug
+        concentration = drug.concentration
+        ret[concentration] = {
+            'name': str(concentration) + drug.concentration_unit,
```

Comment:
Gives me an error: 'ascii' codec can't encode character u'\u03bc' in position 0: ordinal not in range(128)